src/printer.js

```diff
@@ -119,15 +159,15 @@ function printLines(path, options, print) {
     if (lineShouldEndWithSemicolon(stmtPath)) {
       parts.push(";");
     }
-    if (
-      sharedUtil.isNextLineEmpty(text, stmt, options) &&
-      !isLastStatement(stmtPath)
-    ) {
+    if (!isLastStatement(stmtPath)) {
```

Comment:
why split out?

Reply:
@mgrip it is more logic and for future to allow `<?php echo 'test; ?>` in one line/